client/js/autocompletion.js

```diff
@@ -311,8 +311,22 @@ function completeNicks(word, isFuzzy) {
 	return users.filter((w) => !w.toLowerCase().indexOf(word));
 }
 
+function getCommands() {
+	const cmds = constants.commands;
+
+	if (store.state.settings.searchEnabled === false) {
+		const search = cmds.indexOf("/search");
+
+		if (search !== -1) {
+			cmds.splice(search, 1);
```

Comment:
⚠️ This will edit `constants.commands`. We do not copy the list here but just assign the reference to it to another variable. So with the splice on line 321, we remove it from `constants.commands`. What you want is a copy: `const cmds = constants.commands.slice();

Reply:
Ah, I completely missed that I could do the `.slice()` there. I was aware `constants.commands` was being modified and was one of the points that worried me - but yeah... I had to think further about that. Thank you!